installer/mac/SymphonySettingsPlugin/SymphonySettingsPlugin/MyInstallerPane.m

```diff
@@ -24,9 +24,10 @@ - (BOOL)shouldExitPane:(InstallerSectionDirection)dir {
     NSString *podUrl = [_podUrlTextBox stringValue];
     
     // Check if the url contains a protocol, if not, prepend https to it
-    NSString *prefix = @"https://";
-    if (![podUrl hasPrefix:prefix]) {
-        podUrl = [prefix stringByAppendingString:podUrl];
+    NSString *prefix1 = @"https://";
+    NSString *prefix2 = @"http://";
```

Comment:
Change naming please. Should be something like "prefix" and "securePrefix" or something along those lines.